packages/workspace/src/migrations/update-13-10-0/update-tasks-runner.ts

```diff
@@ -0,0 +1,19 @@
+import {
+  readWorkspaceConfiguration,
+  updateWorkspaceConfiguration,
+  Tree,
+} from '@nrwl/devkit';
+
+export function updateTasksRunner(host: Tree) {
+  const config = readWorkspaceConfiguration(host);
+  if (
+    config?.tasksRunnerOptions['default'] &&
+    config?.tasksRunnerOptions['default'].runner ==
+      '@nrwl/workspace/tasks-runner/default'
+  ) {
+    config.tasksRunnerOptions['default'].runner = 'nx/tasks-runner/default';
```

Comment:
@vsavkin migration to 13.10 fails
https://github.com/nrwl/nx/tree/master/packages/nx/tasks-runners

```suggestion
    config.tasksRunnerOptions['default'].runner = 'nx/tasks-runners/default';
```

Reply:
Fixed by https://github.com/nrwl/nx/pull/9778